indico/htdocs/sass/modules/roombooking/_legacy.scss

```diff
@@ -0,0 +1,106 @@
+/* This file is part of Indico.
```

Comment:
True that, but I don't think it's worth figuring out different file names for all this legacy code that is spread around the Defaults.css file...